specifyweb/frontend/js_src/lib/localization/welcome.ts

```diff
@@ -139,29 +139,32 @@ export const welcomeText = createDictionary({
     'en-us': `
       Specify software is a product of the Specify Collections Consortium that
       is governed and funded by its member institutions. Consortium Founding
-      Members include: Commonwealth Scientific and Industrial Research
-      Organisation (CSIRO), Denmark Consortium of Museums, University of
-      Florida, University of Kansas, and University of Michigan. The Consortium
-      operates under the non-profit, 501(c)3, U.S. tax status of the University
-      of Kansas Center for Research. Specify was supported from 1996 to 2018 by
-      grants from the U.S. National Science Foundation.
+      Members include: Commonwealth Scientific and Industrial Research Organisation 
```

Comment:
there are many now, and hard to read - perhaps convert to a bullet list? or should we wait for a few more funding members first? 😊